src/sentry/static/sentry/app/utils/promptsActivity.jsx

```diff
@@ -0,0 +1,13 @@
+import moment from 'moment';
+
+/**
+ * Given a snoozed unix timestamp in seconds,
+ * returns the number of days since the prompt was snoozed
+ *
+ * @param {Number} snoozed_ts Snoozed timestamp
+ */
+export function snoozedDays(snoozed_ts) {
```

Comment:
even though on the backend it's snake_case, let's have this utility function use camelCase.